design-library/src/components/BccReact/BccReact.css

```diff
@@ -13,25 +13,25 @@
     }
     
     .bcc-react-selector {
-        @apply absolute -left-1 z-50 flex items-center gap-1 rounded-full bg-neutral-100 px-1 dark:bg-neutral-600;
+        @apply absolute -left-1 z-50 flex items-center drop-shadow-md gap-1 rounded-full bg-neutral-100 px-1 dark:bg-neutral-600;
         /* Default --bottom */
-        @apply top-11 origin-top-left shadow-lg;
+        @apply top-11 origin-top-left;
     }
     
     .bcc-react-selector--top {
-        @apply -top-10 origin-bottom-left shadow;
+        @apply -top-10 origin-bottom-left;
     }
     
     .bcc-react-selector-item {
         @apply p-2 text-xl leading-none;
     }
 
     .bcc-react-emoji-list-item {
-        @apply flex cursor-pointer items-center justify-center rounded-full p-1 text-2xl leading-none shadow transition-all hover:scale-105;
+        @apply flex cursor-pointer items-center justify-center rounded-full p-1 text-2xl leading-none drop-shadow transition-all hover:scale-105;
         /* Default --not-selected */
-        @apply text-black bg-neutral-100;
+        @apply text-black bg-neutral-100 dark:bg-neutral-800 dark:text-white;
     }
     .bcc-react-emoji-list-item--selected {
-        @apply bg-neutral-800 text-white;
+        @apply bg-neutral-800 text-white dark:bg-polo-blue-800;
```

Comment:
Until now we've not included dark mode for any components, pending the "official" dark mode implementation of the designers. Dark mode for other elements is implemented in the [overrides CSS file](https://github.com/bcc-code/bcc-event-app/blob/master/src/assets/components-overrides.css) in the Event app. For consistency, I feel we should probably keep to that approach for now, and carry these over when implementing dark mode for the whole library.

Reply:
Ok. Updated